src/graph/graph.c

```
@@ -895,6 +895,26 @@ inline bool Graph_EntityIsDeleted
 	return DataBlock_ItemIsDeleted(e->attributes);
 }
 
+// populate 'nodes' with deleted node ids
+void Graph_DeletedNodes
+(
+	const Graph *g,  // graph
+	NodeID **nodes,  // [output] array of deleted node IDs
+	uint64_t *n      // [output] number of deleted node IDs
+) {
+	ASSERT(g     != NULL);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)